eng/sdl-tsa-vars.config

```diff
@@ -10,3 +10,4 @@
 -TsaOnboard $True
 -TsaPublish $True
 -PoliCheckAdditionalRunConfigParams @("UserExclusionPath < $(Build.SourcesDirectory)/s/eng/PoliCheckExclusions.xml")
+-CrScanAdditionalRunConfigParams @("SuppressionsFile < $(Build.SourcesDirectory)/s/CredScanSuppressions.json")
```

Comment:
Do we need to include this in dotnet/runtime as well?

Reply:
Yeah, and you don't need to move the file - just point this towards wherever the file lives (assuming this change is validated by https://dev.azure.com/dnceng/internal/_build/results?buildId=1582945&view=results)

Reply:
Yeah, that is why I was wondering, if it is needed to be specified in this config file we do we need to move it?

Reply:
@wtgodbe it's a nit but could we try to limit our root directory files to those that contributors normally care about❔ I'd rather we left the suppressions file in .config/ or eng/. (eng/ is probably better since .config/ means something special to Arcade's Tools.proj.)